web/modules/custom/ai_screening_project_track/src/Form/ThresholdsForm.php

```diff
@@ -100,7 +102,31 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
     }
 
     $form['#validate'][] = $this->validateThresholds(...);
-    $form['actions'] = [
+
+    $form['form_footer'] = [
+      '#type' => 'container',
+      '#attributes' => ['class' => ['flex', 'justify-between']],
+    ];
+
+    $url = Url::fromUserInput($_SERVER['REQUEST_URI']);
+    $link = new Link($this->t('Go back'), $url);
```

Comment:
Unless I'm missing something, this will add a link to the current page (`$_SERVER['REQUEST_URI']`). You may want to look for the referring page (using https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) instead (if it exists).